pom.xml

```diff
@@ -643,7 +643,7 @@
           <plugin>
             <groupId>org.apache.felix</groupId>
             <artifactId>maven-bundle-plugin</artifactId>
-            <version>3.3.0</version>
+            <version>4.2.1</version>
```

Comment:
This looks straightforward enough, thanks! We'll need to update our build-and-test dependency CQ since this version is not covered in the current one, I'll take care of that.

Also: the bundle plugin is used for more than just the osgi module. I will need to run a couple of local tests to make sure the rest of the build still works as expected.

Reply:
I didn't noticed this fact. Thank you for the additional testing.